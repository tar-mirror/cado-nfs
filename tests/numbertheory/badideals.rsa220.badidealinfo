# bad ideals for poly1=-5731687294699023376520487979709450216291525+630615566214319854581092023097648620621*x+11805000184948451601984776362928407*x^2-21562003075094690166463658097*x^3-99308044351117064845850*x^4+27234181594909908*x^5+102261713400*x^6
# p=2, r=1 : 3 ideals among 6 are bad
# I0:=ideal<O|2,(7+38348142525*alpha+653591126718983722500*alpha^2+8354668560164974471195016437500*alpha^3)/2>; // f=1 e=1
# I1:=ideal<O|2,(31+332350568550*alpha+6045717922150599433125*alpha^2+12532002840247461706792524656250*alpha^3+747567381620133616580203660634924760937500*alpha^4+2730268618800947556015366959195697783341219296875000*alpha^5)/8>; // f=1 e=1
# I2:=ideal<O|2,(27+76696285050*alpha+3431353415274664543125*alpha^2+45950677080907359591572590406250*alpha^3+747567381620133616580203660634924760937500*alpha^4+2730268618800947556015366959195697783341219296875000*alpha^5)/8>; // f=1 e=1
# 3 branches found
2 2 1 1 -2 1 1
2 3 3 1 1 -3 2
2 3 7 1 1 2 -3
# p=2, r=2 : 3 ideals among 6 are bad
# I3:=ideal<O|2,12782714175*alpha+326795563359491861250*alpha^2>; // f=1 e=1
# I4:=ideal<O|2,(5+76696285050*alpha+1143784471758221514375*alpha^2+106795340231447659511457665804989251562500*alpha^4)/2>; // f=1 e=1
# I5:=ideal<O|2,(7+490193345039237791875*alpha^2+106795340231447659511457665804989251562500*alpha^4)/2>; // f=1 e=1
# 4 branches found
2 2 4 1 1 1 1
2 3 10 1 -4 2 2
2 4 22 1 2 -5 3
2 4 30 1 2 3 -5
# p=3, r=2 : 2 ideals among 5 are bad
# I0:=ideal<O|3,(4+238610664600*alpha+2969401662130691480000*alpha^2+66012196030933131624256920000000*alpha^3+150011561587108920904983058689038400000000*alpha^4)/3>; // f=1 e=1
# I1:=ideal<O|3,(10+136348951200*alpha+2969401662130691480000*alpha^2+66012196030933131624256920000000*alpha^3+150011561587108920904983058689038400000000*alpha^4)/3>; // f=1 e=1
# 3 branches found
3 2 5 1 1 1
3 2 2 1 -1 1
3 2 8 1 1 -1
# p=3, r=3 : 2 ideals among 5 are bad
# I3:=ideal<O|3,3+22724825200*alpha+1161939780833748840000*alpha^2+13202439206186626324851384000000*alpha^3>; // f=1 e=1
# I4:=ideal<O|3,2+34087237800*alpha>; // f=1 e=1
# 3 branches found
3 2 9 1 1 1
3 2 12 1 -1 1
3 2 15 1 1 -1
# p=11, r=1 : 2 ideals among 6 are bad
# I1:=ideal<O|11,(124+1617594375600*alpha+10803159119322045000000*alpha^2+116500862867514595180600680000000*alpha^3+1747822706433888558260872894578806400000000*alpha^4+763826259313481400822276010167719289704640000000000*alpha^5)/11>; // f=1 e=1
# I2:=ideal<O|11,(168+1617594375600*alpha+10803159119322045000000*alpha^2+116500862867514595180600680000000*alpha^3+1747822706433888558260872894578806400000000*alpha^4+763826259313481400822276010167719289704640000000000*alpha^5)/11>; // f=1 e=1
# 11 branches found
11 2 12 1 1 1
11 2 23 1 1 1
11 2 34 1 1 1
11 2 45 1 1 1
11 2 56 1 1 1
11 2 67 1 1 1
11 2 89 1 1 1
11 2 100 1 1 1
11 2 111 1 1 1
11 2 1 1 -1 1
11 2 78 1 1 -1
