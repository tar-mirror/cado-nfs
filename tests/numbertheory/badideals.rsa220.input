-ell 1684996666696914987166688442938726917102321526408785780068975640499  -poly badideals.rsa220.poly -badideals badideals.rsa220.badideals -badidealinfo badideals.rsa220.badidealinfo
