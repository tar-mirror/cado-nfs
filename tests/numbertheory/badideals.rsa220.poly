n: 2260138526203405784941654048610197513508038915719776718321197768109445641817966676608593121306582577250631562886676970448070001811149711863002112487928199487482066070131066586646083327982803560379205391980139946496955261
skew: 330880.0
c0: -5731687294699023376520487979709450216291525
c1: 630615566214319854581092023097648620621
c2: 11805000184948451601984776362928407
c3: -21562003075094690166463658097
c4: -99308044351117064845850
c5: 27234181594909908
c6: 102261713400
Y0: -52974922542293279416678326382248522
Y1: 5030366557070505269159
# MurphyE = 7.96e-13
# f(x) = 102261713400*x^6+27234181594909908*x^5-99308044351117064845850*x^4-21562003075094690166463658097*x^3+11805000184948451601984776362928407*x^2+630615566214319854581092023097648620621*x-5731687294699023376520487979709450216291525
# g(x) = 5030366557070505269159*x-52974922542293279416678326382248522
