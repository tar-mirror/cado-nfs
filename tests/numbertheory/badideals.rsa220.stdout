# nmaps0 0
# nmaps1 5
